### ISSUE TITLE
error HTC Desire 626 DEVICE 

### ISSUE THREAD

**Issue by bumbella** [NativeScript/NativeScript#4772]
I get this errors in terminal when I 'tns run android' and application is running.
Device info: android 6.0.1 HTC sense 7.0

```
JS: Angular is running in the development mode. Call enableProdMode() to enable the production mode.
08-30 08:11:13.417 31507 31507 I art     : Rejecting re-init on previously-failed class java.lang.Class<com.android.webview.chromium.TokenBindingManagerAdapter>
08-30 08:11:13.417 31507 31507 I art     : Rejecting re-init on previously-failed class java.lang.Class<com.android.webview.chromium.TokenBindingManagerAdapter>
08-30 08:11:13.437 31507 31507 I art     : Rejecting re-init on previously-failed class java.lang.Class<com.android.webview.chromium.ServiceWorkerControllerAdapter>
08-30 08:11:13.437 31507 31507 I art     : Rejecting re-init on previously-failed class java.lang.Class<com.android.webview.chromium.ServiceWorkerControllerAdapter>
chromium: [INFO:library_loader_hooks.cc(144)] Chromium logging enabled: level = 0, default verbosity = 0
08-30 08:11:13.537 31507 31507 I cr_BrowserStartup: Initializing chromium process, singleProcess=true
chromium: [INFO:CONSOLE(61)] "error 4", source: about:blank (61)
chromium: [INFO:CONSOLE(1)] "Uncaught (in promise) NotSupportedError: The element has no supported sources.", source:  (1)
chromium: [INFO:CONSOLE(58)] "Uncaught TypeError: player.getPlayerState is not a function", source: file:///android_asset/ytb_video.html (58)
chromium: [INFO:CONSOLE(58)] "Uncaught TypeError: player.getPlayerState is not a function", source: file:///android_asset/ytb_video.html (58)
chromium: [INFO:CONSOLE(58)] "Uncaught TypeError: player.getPlayerState is not a function", source: file:///android_asset/ytb_video.html (58)
```

My tns info:
```
All NativeScript components versions information
┌──────────────────┬─────────────────┬────────────────┬──────────────────┐
│ Component        │ Current version │ Latest version │ Information      │
│ nativescript     │ 3.1.1           │ 3.1.3          │ Update available │
│ tns-core-modules │ 3.1.1           │ 3.1.1          │ Up to date       │
│ tns-android      │ 3.1.1           │ 3.1.1          │ Up to date       │
│ tns-ios          │ 3.1.0           │ 3.1.0          │ Up to date       │
└──────────────────┴─────────────────┴────────────────┴──────────────────┘
```

my package json:

```
"dependencies": {
    "@angular/animations": "~4.2.6",
    "@angular/common": "~4.2.6",
    "@angular/compiler": "~4.2.6",
    "@angular/compiler-cli": "~4.2.6",
    "@angular/core": "~4.2.6",
    "@angular/forms": "~4.2.6",
    "@angular/http": "~4.2.6",
    "@angular/platform-browser": "~4.2.6",
    "@angular/platform-browser-dynamic": "~4.2.6",
    "@angular/router": "~4.2.6",
    "fs.realpath": "^1.0.0",
    "nativescript-angular": "~4.2.0",
    "nativescript-barcodescanner": "^2.6.1",
    "nativescript-geolocation": "^3.0.0",
    "nativescript-google-maps-sdk": "^2.1.3",
    "nativescript-pdf-view": "^2.0.1",
    "nativescript-telerik-ui": "~3.0.4",
    "nativescript-theme-core": "~1.0.4",
    "nativescript-zxing": "^1.5.2",
    "reflect-metadata": "~0.1.8",
    "rxjs": "^5.4.2",
    "tns-core-modules": "^3.1.0 || >3.2.0-",
    "zone.js": "^0.8.12"
  },
  "devDependencies": {
    "@angular/compiler-cli": "~4.2.6",
    "@ngtools/webpack": "~1.5.0",
    "babel-traverse": "6.24.1",
    "babel-types": "6.24.1",
    "babylon": "6.17.1",
    "copy-webpack-plugin": "~4.0.1",
    "extract-text-webpack-plugin": "~3.0.0",
    "lazy": "1.0.11",
    "nativescript-css-loader": "~0.26.0",
    "nativescript-dev-android-snapshot": "^0.*.*",
    "nativescript-dev-sass": "^1.2.0",
    "nativescript-dev-typescript": "^0.4.0",
    "nativescript-dev-webpack": "^0.7.3",
    "raw-loader": "~0.5.1",
    "resolve-url-loader": "~2.1.0",
    "sass": "^1.0.0-beta.1",
    "typescript": "~2.4.2",
    "webpack": "~3.2.0",
    "webpack-bundle-analyzer": "^2.8.2",
    "webpack-sources": "~1.0.1"
  },
  "scripts": {
    "ns-bundle": "ns-bundle",
    "publish-ios-bundle": "npm run ns-bundle --ios --publish-app",
    "generate-android-snapshot": "generate-android-snapshot --targetArchs arm,arm64,ia32 --install",
    "start-android-bundle": "npm run ns-bundle --android --run-app",
    "start-ios-bundle": "npm run ns-bundle --ios --run-app",
    "build-android-bundle": "npm run ns-bundle --android --build-app",
    "build-ios-bundle": "npm run ns-bundle --ios --build-app"
  }
```

**Comment by tsonevn** [comment]
Hi @bumbella,
Could you provide some sample project or sample code, which will allow us to reproduce your scenario on our side? The error could be related to the fact that in the project `ytb_video.html` file is used unsupported element tag. You could review your code and validate if similar tags exist. 
Bear in mind that this is only a suggestion and to the sample project will allow me to investigate further the issue.

**Comment by bumbella** [comment]
I use webview component in app. This load a webpage. I think this use ytb_video.html

**Comment by tsonevn** [comment]
Hi @bumbella,
Could you provide a sample code related to the problem?

Thank you in advance for your cooperation.

**Comment by evanjackson09** [comment]
Hi @tsonevn 

I am getting the same errors on a Huawei P8 Lite as well as on an android emulator when using webview. You can recreate the error using the sample code found under 'Basic Web View' in the NativeScript Docs: https://docs.nativescript.org/angular/code-samples/ui/web-view.html.

Alternatively with the following simple code:

HTML:
```
<GridLayout>
        <WebView [src]="webViewSrc"></WebView>
</GridLayout>
```
Component:
```
export class WebViewComponent {
    public webViewSrc: string = "https://google.com";
}
```
I get the following error on Huawei:
```
08-31 12:52:44.060 20640 20640 I art     : Rejecting re-init on previously-failed class java.lang.Class<com.android.webview.chromium.TokenBindingManagerAdapter>
08-31 12:52:44.060 20640 20640 I art     : Rejecting re-init on previously-failed class java.lang.Class<com.android.webview.chromium.TokenBindingManagerAdapter>
08-31 12:52:44.068 20640 20640 I art     : Rejecting re-init on previously-failed class java.lang.Class<com.android.webview.chromium.ServiceWorkerControllerAdapter>
08-31 12:52:44.068 20640 20640 I art     : Rejecting re-init on previously-failed class java.lang.Class<com.android.webview.chromium.ServiceWorkerControllerAdapter>
chromium: [INFO:library_loader_hooks.cc(144)] Chromium logging enabled: level = 0, default verbosity = 0
08-31 12:52:44.213 20640 20640 I cr_BrowserStartup: Initializing chromium process, singleProcess=true
chromium: [ERROR:gl_surface_egl.cc(262)] eglChooseConfig failed with error EGL_BAD_ATTRIBUTE
```

And on the emulator:
```
chromium: [INFO:library_loader_hooks.cc(143)] Chromium logging enabled: level = 0, default verbosity = 0
08-31 06:55:00.808  4019  4019 I cr_BrowserStartup: Initializing chromium process, singleProcess=true
08-31 06:55:01.140  4019  4019 I cr_DRP  : No DRP key due to exception:java.lang.ClassNotFoundException: com.android.webview.chromium.Drp
chromium: [ERROR:gles2_cmd_decoder.cc(2210)] [GroupMarkerNotSet(crbug.com/242999)!:541433C5]GL ERROR :GL_INVALID_OPERATION : BackFramebuffer::Create: <- error from previous GL command
chromium: [ERROR:texture_manager.cc(2942)] [.RenderCompositor-0xc8dcb400]GL ERROR :GL_INVALID_OPERATION : glTexImage2D: <- error from previous GL command
chromium: [ERROR:buffer_manager.cc(438)] [GroupMarkerNotSet(crbug.com/242999)!:545F6EC9]GL ERROR :GL_INVALID_OPERATION : glBufferData: <- error from previous GL command
chromium: [ERROR:gles2_cmd_decoder.cc(2210)] [.RenderWorker-0xc81d8000]GL ERROR :GL_INVALID_OPERATION : GLES2DecoderImpl::DoBindTexImage2DCHROMIUM: <- error from previous GL command
chromium: [ERROR:gles2_cmd_decoder.cc(2210)] [.RenderWorker-0xc81d8000]GL ERROR :GL_INVALID_OPERATION : GLES2DecoderImpl::DoBindTexImage2DCHROMIUM: <- error from previous GL command
```

My tns info is:
```
┌──────────────────┬─────────────────┬────────────────┬─────────────┐
│ Component        │ Current version │ Latest version │ Information │
│ nativescript     │ 3.1.3           │ 3.1.3          │ Up to date  │
│ tns-core-modules │ 3.1.1           │ 3.1.1          │ Up to date  │
│ tns-android      │ 3.1.1           │ 3.1.1          │ Up to date  │
│ tns-ios          │ 3.1.0           │ 3.1.0          │ Up to date  │
└──────────────────┴─────────────────┴────────────────┴─────────────┘
```
My package.json is:
```
"dependencies": {
    "@angular/animations": "~4.2.0",
    "@angular/common": "~4.2.0",
    "@angular/compiler": "~4.2.0",
    "@angular/core": "~4.2.0",
    "@angular/forms": "~4.2.0",
    "@angular/http": "~4.2.0",
    "@angular/platform-browser": "~4.2.0",
    "@angular/router": "~4.2.0",
    "nativescript-angular": "~4.2.0",
    "nativescript-theme-core": "~1.0.4",
    "reflect-metadata": "~0.1.8",
    "rxjs": "~5.4.2",
    "tns-core-modules": "^3.1.1",
    "zone.js": "~0.8.2"
  },
  "devDependencies": {
    "babel-traverse": "6.26.0",
    "babel-types": "6.26.0",
    "babylon": "6.18.0",
    "lazy": "1.0.11",
    "nativescript-dev-typescript": "~0.5.0",
    "typescript": "~2.4.2"
  }
```

**Comment by tsonevn** [comment]
HI @bumbella  @evanjackson09, 
thank you for the provided info. I investigated further this case and was able to reproduce the reported problem only on HTC device. I am attaching sample project, which could be used for debugging.

The problem is reproducible also in a non-Angular project.
[Archive.zip](https://github.com/NativeScript/NativeScript/files/1267436/Archive.zip)

We will research this issue further and will verify, what is causing this problem and if it is something related with the NativeScript or it is something device specific.
for further info, please keep track on the issue.

**Comment by tsonevn** [comment]
HI @bumbella @evanjackson09,

In our further investigation, we found that this error is specific to the device and their Android implementation. Regarding that, this is something that is not related to the NativeScript modules.

I tested the sample project and found also that the WebView component will still load web sites and the content from a local HTML file. For your convenience, I am attaching ScreenShots from the device.

loading `https://google.com` in the WebView
![screenshot_20170901-012648](https://user-images.githubusercontent.com/17448734/29962373-ba162e64-8f0b-11e7-9b06-93967e09db70.png)

loading local  HTML file in the WebView
![screenshot_20170901-012816](https://user-images.githubusercontent.com/17448734/29962376-bd25b64c-8f0b-11e7-9117-f26331888589.png)
